lib/platforms.js

```diff
@@ -6,7 +6,8 @@
  */
 
 module.exports = {
-  osx   : require( './osx' ),
+  //osx   : require( './osx' ),
```

Comment:
@KevinMartin nope, sorry i forgot to uncomment it, I commented it because it makes the tests fail under linux, and I guess under windows too.

I'll add a commit to fix it.